src/content/release-notes/workers-ai.yaml

```
@@ -5,6 +5,11 @@ productLink: "/workers-ai/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2025-03-07"
+    title: Workers AI bug fixes
+    description: |-
+    - We fixed a bug where `max_tokens` defaults were not properly being respected - `max_tokens` now correctly defaults to `256` as displayed on the model pages. If you want to generate more tokens, please set the `max_tokens` parameter to what you need.
```

```suggestion
    - We fixed a bug where `max_tokens` defaults were not properly being respected - `max_tokens` now correctly defaults to `256` as displayed on the model pages. Users relying on the previous behaviour may observe this as a breaking change. If you want to generate more tokens, please set the `max_tokens` parameter to what you need.
```
Added a quick note about this being a breaking change so it's less likely that users skim/ignore it in docs.